mod-source-record-storage-server/src/main/java/org/folio/verticle/consumers/AuthorityLinkChunkConsumersVerticle.java

```diff
@@ -19,8 +18,7 @@ public class AuthorityLinkChunkConsumersVerticle extends AbstractConsumerVerticl
 
   private final AuthorityLinkChunkKafkaHandler kafkaHandler;
 
-  @Value("${srs.kafka.AuthorityLinkChunkConsumer.loadLimit:2}")
-  private int loadLimit;
+  private static final int AUTHORITY_LINK_CHUNK_CONSUMER_LOAD_LIMIT = 1;
```

Comment:
I don't agree with this solution as I don't see the point in it. We always use only 1 worker to process events from Kafka, and parallel processing of records is performed by multiple instances of the module on prod envs.
`new DeploymentOptions().setWorker(true).setInstances(instancesNumber.orElse(1))`

Reply:
@Aliaksandr-Fedasiuk , The PR is in Draft state now and not ready for review yet, I have already changed the implementation and about to update the PR and make it ready for review soon.